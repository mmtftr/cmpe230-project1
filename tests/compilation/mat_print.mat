matrix A[3,2]
vector C[2]

A = { 1 1.2 2 2 3.5 3 }
C = { 9 8 }

print(A)
printsep()
print(C)