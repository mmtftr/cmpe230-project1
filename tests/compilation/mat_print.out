1 1.200000
2 2
3.500000 3
------------
9
8